common/component_interface_specs/include/component_interface_specs/vehicle.hpp

```diff
@@ -0,0 +1,106 @@
+
+#include <autoware_auto_vehicle_msgs/msg/gear_report.hpp>
+#include <autoware_auto_vehicle_msgs/msg/hazard_lights_report.hpp>
+#include <autoware_auto_vehicle_msgs/msg/steering_report.hpp>
+#include <autoware_auto_vehicle_msgs/msg/turn_indicators_report.hpp>
+#include <geometry_msgs/msg/accel_with_covariance_stamped.hpp>
+#include <nav_msgs/msg/odometry.hpp>
+#include <tier4_map_msgs/msg/map_projector_info.hpp>
+#include <tier4_vehicle_msgs/msg/battery_status.hpp>
+
+namespace vehicle_interface
+{
+
+struct KinematicState
+{
+  using Message = nav_msgs::msg::Odometry;
+  static constexpr char name[] = "/localization/kinematic_state";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
+};
+
+struct Acceleration
+{
+  using Message = geometry_msgs::msg::AccelWithCovarianceStamped;
+  static constexpr char name[] = "/localization/acceleration";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
+};
+
+struct SteeringStatus
+{
+  using Message = autoware_auto_vehicle_msgs::msg::SteeringReport;
+  static constexpr char name[] = "/vehicle/status/steering_status";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
+};
+
+struct GearStatus
+{
+  using Message = autoware_auto_vehicle_msgs::msg::GearReport;
+  static constexpr char name[] = "/vehicle/status/gear_status";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
+};
+
+struct TurnIndicatorStatus
+{
+  using Message = autoware_auto_vehicle_msgs::msg::TurnIndicatorsReport;
+  static constexpr char name[] = "/vehicle/status/turn_indicators_status";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
+};
+
+struct HazardLightStatus
+{
+  using Message = autoware_auto_vehicle_msgs::msg::HazardLightsReport;
+  static constexpr char name[] = "/vehicle/status/hazard_lights_status";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
+};
+
+struct MapProjectorInfo
+{
+  using Message = tier4_map_msgs::msg::MapProjectorInfo;
+  static constexpr char name[] = "/map/map_projector_type";
+  static constexpr size_t depth = 1;
+  static constexpr auto reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL;
+};
```

Comment:
Please move this to map.hpp (new file).

Reply:
fixed in [c5d7a70](https://github.com/autowarefoundation/autoware.universe/pull/2930/commits/c5d7a70861bacad66d454f63a8dfbed210af96db)